molo/surveys/models.py

```diff
@@ -366,7 +373,7 @@ def serve_questions(self, request):
             # Create empty form for non-POST requests
             form_class = self.get_form_class_for_step(step)
             form = form_class(page=self, user=request.user)
-        context = self.get_context(request)
+
```

Comment:
did you intentionally leave this line out?

Reply:
I added it on top